source/contracts/Augur.sol

```diff
@@ -3,13 +3,98 @@ pragma solidity 0.4.17;
 
 import 'Controlled.sol';
 import 'libraries/token/ERC20.sol';
+import 'reporting/IUniverse.sol';
 
 
 // AUDIT/CONSIDER: Is it better that this contract provide generic functions that are limited to whitelisted callers or for it to have many specific functions which have more limited and specific validation?
 contract Augur is Controlled {
+    event MarketCreated(address indexed universe, address indexed market, address indexed marketCreator, uint256 marketCreationFee, string extraInfo);
+    event DesignatedReportSubmitted(address indexed universe, address indexed reporter, address indexed market, address stakeToken, uint256 amountStaked, uint256[] payoutNumerators);
+    event ReportSubmitted(address indexed universe, address indexed reporter, address indexed market, address stakeToken, uint256 amountStaked, uint256[] payoutNumerators);
+    event WinningTokensRedeemed(address indexed universe, address indexed reporter, address indexed market, address stakeToken, uint256 amountRedeemed, uint256 reportingFeesReceived, uint256[] payoutNumerators);
+    event ReportsDisputed(address indexed universe, address indexed disputer, address indexed market, uint8 reportingPhase, uint256 disputeBondAmount);
```

Comment:
Why is reportingPhase a uint8 instead of a ReportingPhase?

Reply:
Was following the types provided by @tinybike

Reply:
On the client side, sending through `ReportingPhase` will come through as a uint8 until such time as enums are ABI encodable (something that is in development by the solc team I believe).  Once that is implemented, I think we'll be better off having the actual enum come through as it will allow consumers (particularly automated rendering systems like EtherScan) to be _way_ more intelligent about what they display (`5` is meaningless, but AWAITING FOR MIGRATION is not).

cc @tinybike since you want to weigh in on every change.

Reply:
Actually, maybe it will be uint256... can you find out what ABI solc spits out for an event with an enum parameter in it?

Reply:
uint8. I feel like this is a more than reasonable change given that

Reply:
Ah -- the ABI JSON files unfortunately aren't able to include things like enums, so they just display it as the smallest integer type possible.  In this case the ABI JSON display type is uint8.  I think it's fine to have an enum in the Augur.sol file though -- IIRC the example Augur.sol file I put on the proposed-events branch actually does use enums :grin: